pillarbox-cast/src/main/java/ch/srgssr/pillarbox/cast/PillarboxCastPlayer.kt

```diff
@@ -52,21 +61,20 @@ class PillarboxCastPlayer(
         seekForwardIncrementMs,
         maxSeekToPreviousPositionMs,
     ),
+    private val trackSelector: CastTrackSelector = DefaultCastTrackSelector()
 ) : PillarboxPlayer, Player by castPlayer {
     private val listeners = ListenerSet<Player.Listener>(castPlayer.applicationLooper, Clock.DEFAULT) { listener, flags ->
         listener.onEvents(this, Player.Events(flags))
     }
-    private val remoteClientCallback = RemoteClientCallback()
     private val sessionManagerListener = SessionListener()
-
+    private var trackSelectionParameters: TrackSelectionParameters = TrackSelectionParameters.DEFAULT_WITHOUT_CONTEXT
     private var remoteMediaClient: RemoteMediaClient? = null
         set(value) {
```

Comment:
I think we can remove the custom setter now

Reply:
I don't understand, what do you mean ?